src/web/nextui/src/app/eval/ResultsTable.tsx

```diff
@@ -647,28 +685,38 @@ function ResultsTable({
           ))}
         </thead>
         <tbody>
-          {reactTable.getRowModel().rows.map((row, rowIndex) => {
+          {reactTable.getRowModel().rows.map((row) => {
             let colBorderDrawn = false;
+
             return (
-              <tr key={row.id}>
+              <tr key={row.id} id={`row-${row.id}`}>
                 {row.getVisibleCells().map((cell) => {
                   const isMetadataCol =
                     cell.column.id.startsWith('Variable') || cell.column.id === 'description';
                   const shouldDrawColBorder = !isMetadataCol && !colBorderDrawn;
                   if (shouldDrawColBorder) {
                     colBorderDrawn = true;
                   }
-                  const shouldDrawRowBorder = rowIndex === 0 && !isMetadataCol;
+                  const shouldDrawRowBorder = !isMetadataCol;
                   return (
                     <td
                       key={cell.id}
                       style={{
                         width: cell.column.getSize(),
                       }}
-                      className={`${isMetadataCol ? 'variable' : ''} ${
-                        shouldDrawRowBorder ? 'first-prompt-row' : ''
-                      } ${shouldDrawColBorder ? 'first-prompt-col' : ''}`}
+                      className={`${isMetadataCol ? 'variable' : ''} ${shouldDrawRowBorder ? 'first-prompt-row' : ''} ${shouldDrawColBorder ? 'first-prompt-col' : ''}`}
                     >
+                      {/* Share Button for Metadata Columns */}
```

Comment:
Would it make sense to put this in the group of icons that appear on the row when the user holds down "shift"?  I also think a link icon might be better